lib/Image/Image.js

```diff
@@ -11,26 +11,30 @@ var
 	Img = require('enyo/Image');
 
 var
-	ImageBadge = require('../ImageBadge');
+	Icon = require('../Icon'),
+	Overlay = require('../Overlay');
 
 /**
 * {@link moon.Image} is a simple control that wraps an {@link enyo.Image} to provide proper
 * alignment with text-based controls.
 *
-* In addition, `moon.Image` accepts optional {@link moon.ImageBadge} client components
-* (ImageBadge being the default kind of Image), which are placed inside a container positioned
-* over the image.  These badges are normally persistent, but may be shown or hidden based on
-* {@glossary Spotlight} focus, using the
-* [showBadgesOnSpotlight]{@link moon.Image#showBadgesOnSpotlight} property.
+* In addition, `moon.Image` adds {@link moon.Overlay} support to show controls over the image. This
+* can be used to add action icons as in the example below.
 *
 * ```
-* {kind: 'moon.Image', src: 'assets/movie.png', showBadgesOnSpotlight: true, components: [
-* 	{src: 'assets/icon-recommended.png'},
-* 	{src: 'assets/icon-favorite.png'},
-* 	{src: 'assets/icon-new/png', classes: 'float-right'}
-* ]}
+* {kind: 'moon.Image', src: 'assets/movie.png',
+* 	overlayShowing: 'spotlight', overlayPosition: 'bottom', overlayComponents: [
+* 		{kind: Icon, src: 'assets/icon-recommended.png'},
+* 		{kind: Icon, icon: 'star'},
+* 		{kind: Icon, src: 'assets/icon-new.png'}
+* 	]
+* }
 * ```
 *
+* For backwards compatibility, component configurations specified using the `components` block
+* within an `moon.Image` will be mapped to `overlayComponents` and will default the `kind` to be
+* {@link moon.Icon}. This behavior is considered deprecated and will be removed in a future release.
```

Comment:
If this is deprecated functionality, why ImageBadgeSample.js in the other PR (https://github.com/enyojs/enyo-strawman/pull/22/files#diff-10120f68294655a53b7a059825f90293R25) explicitly being changed _into_ this method, instead of using the newer method?